src/styles/product.scss

```
@@ -0,0 +1,25 @@
+@import "./colors";
+
+.product-box {
+  width: 285px;
+  height: 446px;
+  display: flex;
+  flex-direction: column;
+  background-color: $productBG;
+  text-align: left;
+
+  & img {
```

Example using BEM naming aproach: 
```suggestion
  &__image {  // Result:  .product-box__image
```

---

You’re right, one of the things I struggle with the most is naming stuff. I often don’t know what to call things, and I also have a hard time figuring out when to use 'kebab-case', etc. I definitely need to get better at that and other things.

---

Its completly normal, it just come with practice